cpp/log/tree_signer.cc

```diff
@@ -314,9 +314,11 @@ void TreeSigner::TimestampAndSign(uint64_t min_timestamp,
   sth->set_timestamp(timestamp);
   sth->set_tree_size(cert_tree_->LeafCount());
   LogSigner::SignResult ret = signer_->SignTreeHead(sth);
-  if (ret != LogSigner::OK)
+  if (ret != LogSigner::OK) {
+    LOG(ERROR) << "Log Signer Error.";
```

Comment:
Might be nice to print out the value of `ret` here.

This whole section is really `CHECK_EQ(ret, LogSigner::OK);`, maybe simpler to just change it to that anyway?